code/game/objects/machinery/mortar.dm

```diff
@@ -401,6 +419,7 @@
 	var/deployable_item = /obj/machinery/deployable/mortar
 	resistance_flags = RESIST_ALL
 	w_class = WEIGHT_CLASS_BULKY //No dumping this in most backpacks. Carry it, fatso
+	var/list/linked_binoculars
```

Comment:
Is this used anywhere?
either way the identical names make it confusing which is which + no autodoc

Reply:
renamed var